Docs/reference/content/reference/driver/crud/client_side_encryption.md

```diff
@@ -1,7 +1,7 @@
 
 +++
-date = "2019-09-30T20:38:-04:00"
-title = "Client-Side Field Level Encryption"
+date = "2019-09-30T20:38:42-04:00"
+title = "Client-Side Encryption"
```

Comment:
I had to shorten this title because otherwise it looked like this

Reply:
Note the weird indentation in the navbar

Reply:
![image](https://user-images.githubusercontent.com/33040432/66141945-9a87e180-e5d2-11e9-881d-77e6c742c866.png)

Reply:
That sucks. So we prioritize looks over consistent nomenclature?

Reply:
Should we revert the change and just leave the navbar looking a bit weird? I'm fine with this too